src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.querySelectorAll('tr');
+
+// Отримуємо заголовок таблиці окремо
+const headRow = table.querySelector('thead tr');
+const headCells = headRow.querySelectorAll('th');
+
+// Створюємо нову колонку для кожного рядка (тіло таблиці)
+for (const row of rows) {
+  const cells = row.querySelectorAll('td');
+
+  // Якщо є клітинки в тілі таблиці
+  if (cells.length > 1) {
+    const newCell = document.createElement('td');
+
+    newCell.textContent = cells[1].textContent;
+
+    const lastCell = row.lastElementChild;
+
+    row.insertBefore(newCell, lastCell);
+  }
+
+  // Якщо є клітинки в заголовку таблиці
+  if (row === headRow && headCells.length > 1) {
+    const newHeaderCell = document.createElement('th');
+
+    newHeaderCell.textContent = cells[1] ? cells[1].textContent : '';
+
+    newHeaderCell.textContent = headCells[1].textContent;
```

Comment:
There is a redundancy here. The text content of `newHeaderCell` is being set twice. The first assignment on line 29 is unnecessary and can be removed since it is immediately overwritten on line 31.